libcfnet/client_code.h

```diff
@@ -47,8 +47,23 @@ AgentConnection *ServerConnection(const char *server, const char *port, const Rl
 void DisconnectServer(AgentConnection *conn);
 
 bool CompareHashNet(const char *file1, const char *file2, bool encrypt, AgentConnection *conn);
+
+/**
+ * @brief Copy regular file over network.
+ * @param source The source filename.
+ * @param dest The destination filename.
+ * @param size The source file size.
+ * @param encrypt Whether or not to encrypt the transmission.
+ * @param conn The connection object to communicate with server.
+ * @param mode The source file mode (perms).
```

Comment:
Why is there a parameter for SOURCE file mode? It would make a lot more sense to me if this was the DESTINATION mode.

Reply:
It creates the destination file, based on a remote `stat` of the source file. Extended the description to make this more clear 😉